C-Plus-Plus/sort/bucket_sort.cpp

```diff
@@ -0,0 +1,43 @@
+// Usecase : if all the elements are in a given range 
+// as in the case we perform counting sort.
+// As in this example, assume all elements are between
+// 0 - 99. We select a fair bucket size, as in this case 
+// 10
+// average time complexity : O(nlog(n/k)), where k is #buckets
+// worst case time complexity : O(nlogn)
+// space complexity : O(n)
+
+#include <bits/stdc++.h>
+using namespace std;
+
+void bucket_sort(vector<int>& arr) {
+	int n = arr.size(), k = 10;
```

Comment:
Use meaningful variable names. Code Quality is one of the important aspects here. Try using Snake Case or Camel Case to name your variables.

Reply:
I don't yet see usage of meaningful variable names.